src/revivalpmmp/pureentities/entity/animal/walking/Ocelot.php

```diff
@@ -47,6 +47,7 @@
 
 
 class Ocelot extends WalkingAnimal implements IntfTameable, IntfCanBreed, IntfCanInteract, IntfCanPanic {
+    use Tameable;
```

Comment:
wtf is this here for?

Reply:
Helps reduce duplicate code between Tameable Entities.

Traits
http://php.net/manual/en/language.oop5.traits.php